packages/react-components/react-color-picker-preview/library/src/components/AlphaSlider/useAlphaSliderState.ts

```diff
@@ -5,15 +5,9 @@ import { useFluent_unstable as useFluent } from '@fluentui/react-shared-contexts
 import { alphaSliderCSSVars } from './useAlphaSliderStyles.styles';
 import type { AlphaSliderState, AlphaSliderProps } from './AlphaSlider.types';
 import { useColorPickerContextValue_unstable } from '../../contexts/colorPicker';
-
-const { sliderProgressVar, sliderDirectionVar, thumbColorVar, railColorVar } = alphaSliderCSSVars;
-
-const MIN = 0;
-const MAX = 100;
-
-const getPercent = (value: number, min: number, max: number) => {
-  return max === min ? 0 : ((value - min) / (max - min)) * 100;
-};
+import { MIN, MAX } from '../../utils/constants';
+import { getPercent } from '../../utils/getPercent';
+import type { HsvColor } from '../ColorPicker/ColorPicker.types';
```

Comment:
suggestion: I recommend moving this to `src/types/color.ts` since it is a shared type used across all components within the package.

Reply:
fixed